Scoop/Controller/Profile/EditProfileViewController.swift

```diff
@@ -59,7 +59,12 @@ class EditProfileViewController: UIViewController {
     private var talkative: Float?
     
     // MARK: - Constants
-    
+
+    private let bottomInset = 36
+    private let buttonCornerRadius = CGFloat(25)
+    private let buttonHeight = 50
+    private let buttonLeadingTrailingInset = 24
+    private let buttonWidth = 104
     private let leadingTrailingInset = 32
     private let textFieldHeight = 56
```

Comment:
If this style is used in other places throughout the codebase then it's fine. However, I like to convert it to the following:
```suggestion
    private struct Constants {
        static let bottomInset: CGFloat = 36
        static let buttonCornerRadius: CGFloat = CGFloat(25)
        static let buttonHeight: CGFloat = 50
        static let buttonLeadingTrailingInset: CGFloat = 24
        static let buttonWidth: CGFloat = 104
        static let leadingTrailingInset: CGFloat = 32
        static let textFieldHeight: CGFloat = 56
    }
```
You would then call it using `Constants.bottomInset`